kalite/contentload/management/commands/generate_assessment_zips.py

```diff
@@ -6,17 +6,29 @@
 import tempfile
 import zipfile
 from multiprocessing.dummy import Pool as ThreadPool
+from threading import Lock
 
 from django.conf import settings
 from django.core.management.base import NoArgsCommand
 
 import kalite.version as version
 
+from kalite.topic_tools import get_content_cache, get_exercise_cache
+
 logging = settings.LOG
 
 ZIP_FILE_PATH = os.path.join(settings.PROJECT_PATH, "assessment_item_resources.zip")
 ASSESSMENT_ITEMS_PATH = os.path.join(settings.PROJECT_PATH, "..", "data", "khan", "assessmentitems.json")
 
+IMAGE_URL_REGEX = re.compile('https?://[\w\.\-\/]+\/(?P<filename>[\w\.\-]+\.(png|gif|jpg|jpeg))', flags=re.IGNORECASE)
```

Comment:
Added the `IGNORECASE` here, instead, so I could remove it from `re.sub` below and thereby get 2.6 compatibility.